ml-testing-toolkit/values.yaml

```diff
@@ -109,147 +109,6 @@ connection-manager-frontend:
-      max_user_connections=994
-      thread_cache_size=999
-
-  # metricsExporter:
-  #   enabled: true
-  #   tag: v0.11.0
-  #   commandOverrides: []
-  #   argsOverrides: []
-
-  # logTail: true
-
-  service:
-    type: ClusterIP
-    port: 3306
-    externalIPs: []
-
-
-mongodb:
-  ## repo and documentation https://github.com/bitnami/charts/tree/master/bitnami/mysql
-
-  enabled: false
-  # # For Hosted Mode
-  # enabled: true
-
-  nameOverride: ttk-mongodb
-
-  ## @param architecture MongoDB(&reg;) architecture (`standalone` or `replicaset`)
-  ##
-  architecture: replicaset
-  ## @param useStatefulSet Set to true to use a StatefulSet instead of a Deployment (only when `architecture=standalone`)
-  ##
-  ## MongoDB(&reg;) Authentication parameters
-  ##
-  auth:
-    ## @param auth.enabled Enable authentication
-    ## ref: https://docs.mongodb.com/manual/tutorial/enable-authentication/
-    ##
-    enabled: true
-    ## @param auth.rootUser MongoDB(&reg;) root user
-    ##
-    rootUser: root
-    ## @param auth.rootPassword MongoDB(&reg;) root password
-    ## ref: https://github.com/bitnami/bitnami-docker-mongodb/blob/master/README.md#setting-the-root-password-on-first-run
-    ##
-    rootPassword: adminpass
-    ## MongoDB(&reg;) custom users and databases
-    ## ref: https://github.com/bitnami/bitnami-docker-mongodb/blob/master/README.md#creating-users-and-databases-on-first-run
-    ## @param auth.usernames List of custom users to be created during the initialization
-    ## @param auth.passwords List of passwords for the custom users set at `auth.usernames`
-    ## @param auth.databases List of custom databases to be created during the initialization
-    ##
-    usernames:
-      -   'ttk'
-    passwords:
-      -   'ttk'
-    databases:
-      -   'ttk'
-
-    ## @param auth.replicaSetKey Key used for authentication in the replicaset (only when `architecture=replicaset`)
-    ##
-    replicaSetKey: ""
-    ## @param auth.existingSecret Existing secret with MongoDB(&reg;) credentials (keys: `mongodb-passwords`, `mongodb-root-password`, `mongodb-metrics-password`, ` mongodb-replica-set-key`)
-    ## NOTE: When it's set the previous parameters are ignored.
-    ##
-    existingSecret: ""
-  tls:
-    ## @param tls.enabled Enable MongoDB(&reg;) TLS support between nodes in the cluster as well as between mongo clients and nodes
-    ##
-    enabled: false
-    ## @param tls.autoGenerated Generate a custom CA and self-signed certificates
-    ##
-    autoGenerated: true
-    ## @param tls.existingSecret Existing secret with TLS certificates (keys: `mongodb-ca-cert`, `mongodb-ca-key`, `client-pem`)
-    ## NOTE: When it's set it will disable certificate creation
-    ##
-    existingSecret: ""
-    ## Add Custom CA certificate
-    ## @param tls.caCert Custom CA certificated (base64 encoded)
-    ## @param tls.caKey CA certificate private key (base64 encoded)
-    ##
-    caCert: ""
-    caKey: ""
 
 keycloak:
```

Comment:
Keycloack is something else that we can look at, but at the moment I believe the TTK uses its own "internal" definitions for Keycloack.

So this would NOT be considered an "external dependency", although it should be.

This something that we should think about a bit more and also verify with @vijayg10 .

Reply:
As discussed, revert config as this will be done/tested as part of https://github.com/mojaloop/project/issues/3091

Reply:
reverted